zio-http/src/main/scala/zio/http/netty/server/ServerInboundHandler.scala

```diff
@@ -175,11 +176,10 @@ private[zio] final case class ServerInboundHandler(
       case _                    => throw new IllegalArgumentException(s"Unsupported HTTP version: ${nettyHttpVersion}")
     }
 
-    // TODO: We need to bring this back, probably not part of Request.
-    // val remoteAddress = ctx.channel().remoteAddress() match {
-    //   case m: InetSocketAddress => Some(m.getAddress)
-    //   case _                    => None
-    // }
+    val remoteAddress = ctx.channel().remoteAddress() match {
```

Comment:
@scottweaver do you remember was this commented out?

Reply:
Ah I see. The performance reports in the checks below looks good, so I think this is safe to add back